### PR TITLE
PUBD-396 Add editors and advisors to lists and search results

### PR DESCRIPTION
**Pull request by hardyoyo**
Add editors and advisors to lists and search results. Re-use author markup for editors and advisors. May need further design refinement.

### CODE REVIEW

**Review by myucekul — COMMENTED** [review]
The changes look good. I noticed that the labels are all singular (author, editor etc.) but the expectation is that each one can be a list. I assume the labels are as per Program Managers ask.

**Comment by hardyoyo** [comment]
> The changes look good. I noticed that the labels are all singular (author, editor etc.) but the expectation is that each one can be a list. I assume the labels are as per Program Managers ask.

Screenshot_20220715-204835-885

File: app/jsx/components/AuthorListComp.jsx [hunk 1]
```diff
@@ -17,42 +16,60 @@ class AuthorListComp extends React.Component {
...
+      } 
     })
   }
 
   render() {
     let p = this.props,
-        year = p.pubdate.match(/\d{4}/),
-        source = p.source,
+        year = p.pubdate ? (p.pubdate.match(/\d{4}/)) : null,
+        source = p.source ? p.source : null,
```
Comment (myucekul) [inline app/jsx/components/AuthorListComp.jsx:34]: Isn't 'source = p.source ? p.source : null' same as 'source = p.source'

**Review by myucekul — APPROVED** [review]

File: app/jsx/components/AuthorListComp.jsx (hunk 1 above)
Comment (hardyoyo) [reply app/jsx/components/AuthorListComp.jsx:34]: you're right, I was trying to rule out some possibilities, but I'll fix this

**Pull request merged**